music21/search/serial.py

```diff
@@ -655,7 +655,7 @@ def searchIncludeAllInclude(self, n, partNumber):
 
         chordList = self.chordList
         chordList.append(n)
-        self.totalLength = self.totalLength + len(n.pitches)
```

Comment:
OH!  I remember why you'll find these!  There was a bug in some editor once that for some reason flagged += on self attributes with camelcases as not allowed.  Thank goodness that's gone!